python/neuroglancer/ingest/volumes/gcloudvolume.py

```diff
@@ -5,52 +5,43 @@
 import numpy as np
 from tqdm import tqdm
 
-import lib
-from lib import clamp, xyzrange, Vec3, Bbox, min2
 import neuroglancer
-from volumes import Volume
-from google.cloud import storage as gstorage
+import neuroglancer.ingest.lib as lib
+from neuroglancer.ingest.lib import clamp, xyzrange, Vec3, Bbox, min2
+from neuroglancer.ingest.volumes import Volume
+from neuroglancer.ingest.base import Storage
 
 class GCloudVolume(Volume):
-  def __init__(self, dataset_name, layer, mip, cache_files=True, use_ls=True):
-    super(self.__class__, self).__init__()
 
-    self.dataset_name = dataset_name
-    self.layer = layer
+  def __init__(self, dataset_name, layer_name, mip=0, cache_files=True, use_ls=True):
+    self._dataset_name = dataset_name
+    self._layer_name = layer_name
     self.mip = mip
-    
-    self.cache_files = cache_files
-    self.use_ls = use_ls
-
-    blob = self.__getInfoBlob()
-
-    self.info = json.loads(blob.download_as_string())
+    self._cache_files = cache_files
+    self._use_ls = use_ls
 
-  def __getInfoBlob(self):
-    client = gstorage.Client(project=lib.GCLOUD_PROJECT_NAME)
-    bucket = client.get_bucket(lib.GCLOUD_BUCKET_NAME)
+    self._storage = Storage(dataset_name, layer_name) 
+    self._get_info()
 
-    info_path = os.path.join(self.dataset_name, self.layer, 'info')
-    return gstorage.blob.Blob(info_path, bucket)
+  def _get_info(self):
+    self._info_blob =  self._storage.get_blob(
+                '{}/{}/info'.format(self._dataset_name, self._layer_name))
+    self.info = json.loads(self._info_blob.download_as_string())
 
-  def addScale(self, factor):
+  def add_scale(self, factor):
```

Comment:
I read PEP8. We can do this, but I think GVR is wrong. He doesn't want to use mixed case for anything and wants to use CamelCase for object class names. My feeling is that this is incorrect because within a class, one must distinguish between nested functions, public data members (sometimes aliased as properties) and method names. GVR recommends using function case for all functions, but this means when looking at a dir(...) that it's not possible to tell at a glance which are data and which are methods.